sycl/test/basic_tests/nd_item.cpp

```diff
@@ -1,107 +0,0 @@
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
Do we have an existing  test for the new nd_item?

Reply:
It can't be unit-tested that way anymore - the fix in https://github.com/intel/llvm/pull/12236 changed these function to delegate to SPIR-V intrinsics instead of storing the state internally.

Reply:
Ok, got it